packages/fxa-payments-server/src/styles/_mixins.scss

```diff
@@ -37,7 +36,7 @@
 
   p {
     margin: 0;
-    color: $grey-40;
+    color: $grey-50;
```

Comment:
`$grey-50` is #737373, though Jess' spec actually specifies a couple of other hex color codes that don't match any of our grey variables. I'm guessing this specific thing happened because the comments in the spec call out Grey-40, Grey-50 and so on, but their actual hex values don't match the greys in our Photon package. CC @jess-cook03